cluster/addons/dns/coredns/coredns.yaml.base

```diff
@@ -109,8 +109,6 @@ spec:
         image: k8s.gcr.io/coredns:1.1.3
         imagePullPolicy: IfNotPresent
         resources:
-          limits:
```

Comment:
We have a couple of reports of issues with the default limits.  This needs to be addressed imo